src/gui/qgssublayersdialog.h

```diff
@@ -59,10 +88,21 @@ class GUI_EXPORT QgsSublayersDialog : public QDialog, private Ui::QgsSublayersDi
     typedef QList<QgsSublayersDialog::LayerDefinition> LayerDefinitionList;
 
     //! Constructor for QgsSublayersDialog
+
+    /**
+     * Construct a new QgsSublayersDialog object - a dialog to select which sub layers to be imported from a file (e.g. from geopackage or zipfile)
```

Comment:
Note that this dialog can be used for non-file based sources too.
```suggestion
     * Construct a new QgsSublayersDialog object - a dialog to select which sub layers to be imported from a data source (e.g. from geopackage or zipfile)
```

Reply:
Done!